llvm/include/llvm/BinaryFormat/ELFRelocs/RISCV.def

```diff
@@ -56,3 +56,7 @@ ELF_RELOC(R_RISCV_IRELATIVE,         58)
 ELF_RELOC(R_RISCV_PLT32,             59)
 ELF_RELOC(R_RISCV_SET_ULEB128,       60)
```

Comment:
Define the dynamic relocation type R_RISCV_TLSDESC  12 as well.

(The lld patch incorrectly uses `R_RISCV_TLSDESC_CALL` for the dynamic reloc. I am going to take over that part and TLS optimization.)

Reply:
I've removed them per your request. Just so we'd have a version where they were in place, I added the code to make them relaxable. But I'm fine either way, so if you'd prefer we start w/o the relaxation relocations it should be good now.

Related: do you think its worth testing that they're *not* there?

Reply:
> Related: do you think its worth testing that they're not there?

I've made a comment on the test "I think we should use llvm-objdump -d -r to check relocations."
If you do that, `-NEXT` is sufficient to ensure that there is no `R_RISCV_RELAX` :)

With my experience on https://github.com/llvm/llvm-project/pull/79099 , I think two `R_RISCV_RELAX` are sufficient. I probably need to follow up on the spec.